core-modules/be-core-modules/data-bases/relationalDataBase/models/prod/Question.js

```diff
@@ -10,11 +10,15 @@ module.exports = function(sequelize, DataTypes) {
       type: DataTypes.STRING,
       allowNull: false
     },
-    department: {
+    hierarchy: {
       type: DataTypes.STRING,
       allowNull: false
     },
-    description: {
+    organization: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    decision: {
```

Comment:
what do you mean by decision?

Reply:
Is de decision you have to answered by choising a response

Reply:
I don't quite understand it. We can talk about it through a phone call

Reply:
fixed